packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -561,6 +561,8 @@ export const messages = {
   conflict_detect_diff_command_title: 'Compare Files',
 
   force_source_diff_text: 'SFDX: Diff File Against Org',
+  force_source_diff_remote_not_found:
+    'Selected file or directory was not found in org',
```

Comment:
@sbudhirajadoc Could I have a review on the text? This is shown when the remote file diffed is not found.

Reply:
change to 'Selected file or directory was not found in the org'